include/openssl/ssl.h

```diff
@@ -262,6 +262,9 @@ typedef int (*custom_ext_parse_cb) (SSL *s, unsigned int ext_type,
                                     const unsigned char *in,
                                     size_t inlen, int *al, void *parse_arg);
 
+/* Typedef for verification callback */
+typedef int (*verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
```

Comment:
it IS API/ABI compatible, as typedef doesn't introduce changes...
you like it.... you really like it...  you feel your eyes are getting heavy... you are ready to press plus-one...  i am your father ...    :)